### PR TITLE
Preloaded Level

### LINKED ISSUE #30: Create testing level
Create a small scale testing level to test game object interactions

### PR DESCRIPTION
**Pull request by ruihan00**
- fix bug in running animation by setting friction of all physics body to zero for now
- added preloaded level with various game object for testing
- temporarily set floor to 4000 width,  will  need to set to size of level instead in another pr
- set obstacle,. tool to be non dynamic, 
- set tool to disappear on contact
- move event to increase velocity up to certain threshold
Bugs:
- ~~Unable to click on jump when moving~~ Fixed in latest commit
- ~~Collision between other game objects not correctly handled, player pushes obstacles, monsters, etc~~ Minimally playable state now

closes #30

### CODE REVIEW

File: star-dash/star-dash/GameEngine/Collision/CollisionHandler.swift [hunk 1]
```diff
@@ -17,7 +17,8 @@ struct CollisionHandler {
     }
 
     static func between(player: Player, obstacle: Obstacle) -> Event? {
-        nil
+        print("Obstacle contact")
```
Comment (Junyi00) [inline star-dash/star-dash/GameEngine/Collision/CollisionHandler.swift:20]: Could remove the prints

Comment (Junyi00) [inline star-dash/star-dash/GameEngine/Collision/CollisionHandler.swift:20]: Could remove the prints

File: star-dash/star-dash/GameEngine/Entities/GameEntities/Monster.swift [hunk 2]
```diff
@@ -29,12 +29,18 @@ class Monster: Entity {
...
         let healthComponent = HealthComponent(entityId: self.id, health: self.health)
         let physicsComponent = PhysicsComponent(entityId: self.id, size: PhysicsConstants.Dimensions.monster)
+        let spriteComponent = SpriteComponent(entityId: self.id,
+                                              image: self.sprite,
+                                              textureSet: nil,
+                                              textureAtlas: nil,
+                                              size: self.size)
         physicsComponent.collisionBitMask = PhysicsConstants.CollisionMask.monster
-        physicsComponent.affectedByGravity = true
+        physicsComponent.affectedByGravity = false
```
Comment (Junyi00) [inline star-dash/star-dash/GameEngine/Entities/GameEntities/Monster.swift:38]: Why should they not be affected by gravity?

**Review by HoJunHao2000 — APPROVED** [review]
I wouldn't say the running and jumping is completely smooth yet.
Inputs when moving left and jumping still kinda wonky.

I would say keep those (#22 #21) as an open issues/update those issues.

But other than that the preloaded level looks good!

**Pull request merged**